tests/base/pom.xml

```diff
@@ -676,6 +676,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.jboss.xnio</groupId>
+            <artifactId>xnio-api</artifactId>
+            <version>3.7.2.Final</version>
```

Comment:
Even for test scope the component and version should be defined in the wildfly-elytron-parent pom so we have just one place to look when managing versions.

Reply:
Moved it to the parent pom